azkaban-web-server/src/main/java/azkaban/imagemgmt/servlets/HPFlowServlet.java

```diff
@@ -0,0 +1,129 @@
+package azkaban.imagemgmt.servlets;
+
+import azkaban.imagemgmt.daos.HPFlowDao;
+import azkaban.imagemgmt.dto.HPFlowDTO;
+import azkaban.imagemgmt.exception.ErrorCode;
+import azkaban.imagemgmt.exception.ImageMgmtException;
+import azkaban.imagemgmt.exception.ImageMgmtInvalidInputException;
+import azkaban.imagemgmt.exception.ImageMgmtInvalidPermissionException;
+import azkaban.imagemgmt.exception.ImageMgmtValidationException;
+import azkaban.imagemgmt.permission.PermissionManager;
+import azkaban.imagemgmt.services.HPFlowService;
+import azkaban.imagemgmt.utils.ConverterUtils;
+import azkaban.server.HttpRequestUtils;
+import azkaban.server.session.Session;
+import azkaban.user.User;
+import azkaban.webapp.AzkabanWebServer;
+import azkaban.webapp.servlet.LoginAbstractAzkabanServlet;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.http.HttpStatus;
+import org.apache.log4j.Logger;
+
+
+/**
+ * This class handles all the REST API calls for API "/hpFlows"
+ * It currently implements addition of high priority flows.
+ */
+public class HPFlowServlet extends LoginAbstractAzkabanServlet {
+
+  private static final Logger log = Logger.getLogger(HPFlowServlet.class);
+  private static final String BASE_HP_FLOW_URI = "/hpFlows";
+  private ConverterUtils converterUtils;
```

Comment:
Util should not be an instance. All the methods in utils should be public static. Can you please remove this instance?

Reply:
+1 with @jakhani

Reply:
Good call.

Reply:
Utils should only have static methods.